new OpenPOWER GPU request form

leian7 opened the pull request:
For #159 

Changes: 
- [x] new OpenPOWER GPU request form based on powerdev form. In the menu, I put it in Services/PowerLinux/OpenPOWER Development Hosting/ , which is where the powerdev form is. Should it be one level up?

TODO:
- [x] new description for the form
- [x] verify correct email? ~~Currently we have powerdev-request@osuosl.org specified at the bottom of the form, which is also where the form currently routes its email to.~~ openpower-gpu-support@osuosl.org

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

ramereth commented:
@leian7 please use openpower-gpu-support@osuosl.org. I'll ask about a description.

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

leian7 commented:
@ramereth thanks. I've updated the email address. 

For formsender settings, is there already an email key in production's conf.py that corresponds to openpower-gpu-support@osuosl.org? For now I have removed the ``send_to`` field.

ramereth commented:
@leian7 nope, we need to add one for it. Can you please do that?

leian7 commented:
@ramereth Made a merge request on Ansible: https://git.osuosl.org/osuosl/ansible_playbooks/merge_requests/23

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

ramereth commented on content/forms/openpower-gpu-request.rst:
```diff
@@ -0,0 +1,104 @@
+
+          <p><i>You should receive an automated email from our request ticketing system to the email address you have provided
+          within 5-10 minutes.  If you don't receive this email please reach out to us at <a href="mailto:openpower-gpu-support@osuosl.org">openpower-gpu-support@osuosl.org</a> or
+          via IRC in <b>#osuosl</b> on Freenode.</i></p>
```
We should ask Chris or Ian about a different IRC channel or other form of communication.

Kennric replied:
@ramereth  Any updates on this? I think the formsender portion is ready to merge, so this is just needs to have the string composition field added, and the wording correct.

ramereth replied:
None yet unfortunately.

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org

osuosl-manatee commented:
This PR has been automatically built and can be viewed here: http://osuosl-pelican-168.staging.osuosl.org
